ceph_deploy/hosts/centos/__init__.py

```diff
@@ -18,6 +18,13 @@ def choose_init(module):
 
     Returns the name of a init system (upstart, sysvinit ...).
     """
+    with open("/etc/rpm/macros.dist", 'r') as file:
```

Comment:
i don't think we should rely on an rpm macro for determining the distro's release version. user does not necessarily have this package installed.